charts/microgateway/README.md.gotmpl

```diff
@@ -376,6 +377,21 @@ Finally, apply the Helm chart configuration file with `-f` parameter.
   helm upgrade -i microgateway airlock/microgateway -f custom-values.yaml -f env-variables.yaml
   ```
 
+## Extra Volumes
+The Helm Chart allows you to define extra volumes which can then be used in the Microgateway. 
```

Comment:
> The Helm Chart allows you to define extra volumes which can then be used in the Microgateway. 
* chart is always written in lowercase. See other examples in the readme. Please change.
* I would cross out "then"

=> The Helm chart allows you to define extra volumes which can be used in the Microgateway.